schemas/json/layout/expression.schema.v1.json

```
@@ -190,6 +191,17 @@
       ],
       "additionalItems": false
     },
+    "func-roles": {
+      "title": "Roles lookup function",
+      "description": "This function will look up a role that the user has for their current party",
+      "type": "string",
+      "items": [
+        { "const":  "externalApi" },
```

```suggestion
        { "const": "roles" },
```

This should match the function name.